mm2src/coins/z_coin.rs

```diff
@@ -616,6 +625,132 @@ async fn sapling_state_cache_loop(coin: ZCoin) {
     }
 }
 
+struct TransactionShielded {
+    id_out: i32,
+    out_index: i32,
+    tx_hash: H256,
+    out_amount: i64,
+}
+
+fn init_shielded_db(sql: &Connection) -> Result<(), SqliteError> {
+    const INIT_SAPLING_CACHE_TABLE_STMT: &str = "CREATE TABLE IF NOT EXISTS z_shielded (
```

Comment:
The const name is incorrect, it inits z_shielded table. Also it will be more clear to name this table “shielded_dex_fees”.